governance/votes/EXERCISE - Executive vote - July 12, 2023.md

```diff
@@ -0,0 +1,214 @@
+
+- CRVV1ETHSTETH-A [Debt Ceiling](https://manual.makerdao.com/parameter-index/vault-risk/param-debt-ceiling) `(line)` will be set to **zero Dai**.
+- CRVV1ETHSTETH-A will be removed from [Debt Ceiling Instant Access](https://manual.makerdao.com/module-index/module-dciam) (`MCD_IAM_AUTO_LINE`).
+
+### Ecosystem Actor Funding
+
+#### A) Ecosystem Actor Dai Budget Streams
+
+Per the [Support Scope](https://mips.makerdao.com/mips/details/MIP106#7-4-2) and [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108#9-launch-project), the proposed actions concern setting up Dai budget streams to fund the work of Ecosystem Actors Jetstream and Chronicle Labs. 
+
+- [Jetstream](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-jetstream/21054) takes over the existing duties of the Development & UX (DUX-001) Core Unit and will be initially involved in the [Launch Project](http://forum.makerdao.com/t/the-launch-project-and-scope-artifact-update-proposal/21048?u=le_bateleur). The Accessibility Scope Facilitator has [approved](http://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306/2?u=le_bateleur) the onboarding of Jetstream as an Ecosystem Actor.
+
+- Spinning off from the Oracles Core Unit, [Chronicle Labs](https://forum.makerdao.com/t/professional-ecosystem-actor-introduction-chronicle-labs/21053) will launch a decentralized oracle protocol and infrastructure services to support the soon-to-launch SubDAOs. In a recent [poll](https://vote.makerdao.com/polling/QmdnSKPu), Maker governance ratified amendments to the Support Scope that include a [provision incubating Chronicle Labs](https://mips.makerdao.com/mips/details/MIP106#7-4-2-1a) with a one-year linear stream.  
+
+If this executive proposal passes, the following Dai budget streams will be executed:
+
+| Ecosystem Actor               | Start Date | End Date   | Address                                                      | Amount (DAI)  |
+| ----------------------------- | ---------- | ---------- | ------------------------------------------------------------ | ------------- |
+| Jetstream Auditor Wallet      | 2023-07-01 | 2024-12-31 | [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962) | 2,964,006 DAI |
```

Comment:
Potentially include duration in here. It's generally easier for humans to parse than two dates.